general/nuget-packages.md

```diff
@@ -23,7 +23,7 @@ packages.
 | [Proj0213](../rules/Proj0213.md) | Consider adding the icon url for maximum compatibility.
 | [Proj0214](../rules/Proj0214.md) | Do include the package id.
 | *none*                           | Consider choosing a NuGet package name with a prefix that meets NuGet's [prefix reservation criteria](https://learn.microsoft.com/nuget/nuget-org/id-prefix-reservation).
-|                                  | Do use an image that is 128x128 and has a transparent background (PNG) for best viewing results.
+| [Proj0215](../rules/Proj0215.md) | Do use an image that is 128x128 and has a transparent background (PNG) for best viewing results.
```

Comment:
Shouldn't this be above *none* as it is the next rule after 0214?

Reply:
No, I'd like to keep it in order with how Microsoft stated it.